tests/Nox.Types.Tests/Types/CountryCode2/CountryCode2Tests.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Nox.Types.Tests.Types.CountryCode3;
+﻿using Nox.Types.Tests.Types.CountryCode2;
```

Comment:
In other tests the namespace is Nox.Types.Tests.Types. Can we update this one as well?